apps/frontend/src/app/backups-overview/backups/backups/backups.component.ts

```diff
@@ -112,59 +128,142 @@ export class BackupsComponent implements AfterViewInit {
+    // LineSeries hinzufügen
     let series = chart.series.push(
-      am5xy.ColumnSeries.new(root, {
-        name: 'Size',
+      am5xy.LineSeries.new(this.root, {
+        name: 'Backup Size',
         xAxis: xAxis,
         yAxis: yAxis,
         valueYField: 'sizeMB',
-        categoryXField: 'date',
-        tooltip: am5.Tooltip.new(root, {
-          labelText: '{valueY} MB',
+        valueXField: 'creationDate',
+        tooltip: am5.Tooltip.new(this.root, {
+          labelText: 'Size: {valueY} MB\nDate: {valueX}',
+        }),
+      })
+    );
+
+    // Add bullet points - Fixed type definition
+    series.bullets.push((root: am5.Root) => {
+      return am5.Bullet.new(root, {
+        sprite: am5.Circle.new(root, {
+          radius: 5,
+          fill: series.get('fill'),
+          stroke: root.interfaceColors.get('background'),
+          strokeWidth: 2,
         }),
+      });
+    });
+
+    // Add cursor
+    chart.set(
+      'cursor',
+      am5xy.XYCursor.new(this.root, {
+        behavior: 'zoomX',
       })
     );
 
-    //series.data.setAll(chartData);
-    chartData$.subscribe((chartData) => series.data.setAll(chartData));
+    // Nach der Cursor-Definition
+    let scrollbarX = chart.set(
+      'scrollbarX',
+      am5.Scrollbar.new(this.root, {
+        orientation: 'horizontal',
+      })
+    );
+
+    let rangeDate = xAxis.makeDataItem({});
+    let rangeTime = Date.now();
+    rangeDate.set('value', rangeTime);
+    rangeDate.set('endValue', rangeTime - 1000 * 60 * 60 * 24 * 30); // 30 Tage
+
+    xAxis.createAxisRange(rangeDate);
+
+    chartData$.subscribe({
+      next: (data) => {
+        if (data && data.length > 0) {
+          series.data.setAll(data);
 
-    // Animation für Balken
+          // Set zoom to match selected time range
+          const endDate = new Date();
+          const startDate = new Date();
+
+          switch (this.selectedTimeRange) {
+            case 'week':
+              startDate.setDate(startDate.getDate() - 7);
+              break;
+            case 'month':
+              startDate.setMonth(startDate.getMonth() - 1);
+              break;
+            case 'year':
+              startDate.setFullYear(startDate.getFullYear() - 1);
+              break;
+          }
+
+          xAxis.zoomToDates(startDate, endDate);
+        } else {
+          console.warn('No data available for chart');
+        }
+      },
+      error: (error) => {
+        console.error('Error loading chart data:', error);
+      },
+    });
+
+    // Make stuff animate on load
     series.appear(1000);
     chart.appear(1000, 100);
   }
+
+  refresh(option?: ClrDatagridStateInterface<Backup>): void {
+    let params: HttpParams = new HttpParams();
+
+    if (option) {
+      if (option.page) {
+        if (option.page.current && option.page.size)
+          params = params.append(
+            'offset',
+            (option.page.current - 1) * option.page.size
+          );
```

Comment:
Pagination has weird behavior.
On page load I cannot go to next page. There is no button. When I click on one the buttons for selecting the range of the diagram, the buttons for changing page suddenly appear.
On clicking on the next page button the buttons for changing the page disappear again